changelog.dd

```diff
@@ -227,6 +229,27 @@ assert([4, 7, 5].enumerate.maxElement!`a.value` == tuple(1, 7));
 -------
 )
 
+$(LI $(LNAME2 quantize, Added `std.math.quantize`.)
+    $(P $(XREF math, quantize) rounds to the nearest multiple of some number.
+        Features:)
+    $(UL
+        $(LI The rounding method can be specified using the `rfunc` parameter. By
+            default, the current rounding mode will be used, which is typically
+            "banker's rounding".)
```

Comment:
Also, does the notion of "current" rounding mode preclude purity of this function? If so, it would be great if a pure overload would be added.